specification.md

```diff
@@ -2343,7 +2344,9 @@ should download the file:
   and the file SHOULD be downloaded from the location specified in the
   [download_url](#downloadablefiledownload_url) field of the
   [DownloadableFile](#downloadablefile-message) message. The Agent SHOULD use an
-  HTTP GET message to download the file.
+  HTTP GET message to download the file. The Agent SHOULD include the HTTP
```

Comment:
Let's mark this addition as `[Development]`. See how it's done in other places.

Reply:
Apologies for missing that 🤦

Done, both in the proto definition, and in specification (both inline to the message and in the separate section).